bin/exec.go

```diff
@@ -216,6 +218,8 @@ func main() {
 		if err != nil {
 			log.Info("Unable to Update ChaosResult")
 			log.Error(err)
+		} else {
+			analytics.TriggerAnalytics(engineDetails.Experiments[i])
```

Comment:
Why do we have it here ? We should be updating experiment counter irrespective of whether or not runner is able to update result. Would ideally do this before we start the experiment job?

Reply:
Changed the place of execution @ksatchit as requested